examples/react-fcl-demo/flow.json

```diff
@@ -0,0 +1,10 @@
+{
+	"accounts": {
+		"service": {
+			"address": "f8d6e0586b0a20c7",
+			"privateKey": "6933b0c557a30513adb402be741464601667f898e61017e58a0be94f593e0f6f",
```

Comment:
There is a `flow.json` file we use in the root directory. It uses the Private Key we use for testing and things like that: `84f82df6790f07b281adb5bbc848bd6298a2de67f94bdfac7a400d5a1b893de5`. Any chance you can either:
- remove this file and mention the usage of the `flow.json` in the root of the repo
- change the private key and address (address already looks good) to match those in the root `flow.json` file

Reply:
I will go the second route, I want this package to be self contained as much as possible.